packages/components/src/components/post-breadcrumb-item/post-breadcrumb-item.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component, Element, h, Host, Prop, Watch } from '@stencil/core';
+import { version } from '@root/package.json';
+import { checkEmptyOrUrl } from '@/utils';
+
+/**
+ * @slot default - Slot for placing the text inside the breadcrumb item.
+ */
+@Component({
+  tag: 'post-breadcrumb-item',
+  styleUrl: 'post-breadcrumb-item.scss',
+  shadow: true,
+})
+export class PostBreadcrumbItem {
+  @Element() host: HTMLPostBreadcrumbItemElement;
+
+  /**
+   * The optional URL to which the breadcrumb item will link.
+   */
+  @Prop() url?: string | URL;
+
+  private validUrl?: string;
+
+  @Watch('url')
+  validateUrl() {
+    try {
+      this.validUrl = this.constructUrl(this.url);
+    } catch (error) {
+      console.error(error);
```

Comment:
Can you just check if the error is logged twice and if so just remove this line?

Reply:
Will do.